tfrobot/pkg/deployer/deployer.go

```diff
@@ -97,15 +98,20 @@ func RunDeployer(ctx context.Context, cfg Config, tfPluginClient deployer.TFPlug
 	endTime := time.Since(deploymentStart)
 
 	// load deployed deployments
-	outputBytes, err := loadAfterDeployment(ctx, tfPluginClient, passedGroups, failedGroups, cfg.MaxRetries, filepath.Ext(output) == ".json")
-	if err != nil {
-		return err
+	outputBytes, errs := loadAfterDeployment(ctx, tfPluginClient, passedGroups, cfg.MaxRetries, filepath.Ext(output) == ".json")
+	if errs != nil {
+		failedGroupsErr = multierror.Append(failedGroupsErr, errs.Errors...)
```

Comment:
why we don't return here?

Reply:
The load fails if some of the deployments failed, so maybe some deployments return, and we need to print their output before returning, and same for the load